lib/trino-hdfs/src/test/java/io/trino/hdfs/TestFSDataInputStreamTail.java

```diff
@@ -70,6 +70,7 @@ public void tearDown()
         closeAll(
                 () -> fs.delete(new Path(tempRoot.toURI()), true),
                 fs);
+        fs = null;
```

Comment:
why is this a related change ? Can we keep this in a separate commit ?

Reply:
This came in as part of bringing `trino-testing-services` dependency into `trino-hdfs`. Resource deallocation check added via `ManageTestResources` (in PR #15165) got activated in `lib/trino-hdfs` resulting in this test failure here. Adding `fs = null;` fixes this, as was the case with similar changes in the PR referenced. Can move this to a different commit